### ISSUE TITLE
a11y improvements for SocialBanner CommunityBanner ExploreBanner

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1126]
SocialBanner, CommunityBanner, and ExploreBanner accessibility can improve in a few places: 

## SocialBanner

- Social icons missing SVG alts/aria-hidden   
- Social icons missing link text
- Correct Slack link on icon

## CommunityBanner

- SVG missing alt/aria-hidden

## ExploreBanner

- Missing label input fields
